website/content/docs/discovery/checks.mdx

```diff
@@ -413,16 +413,20 @@ key in your configuration file.
 }
 ```
 
-## Success/Failures before passing/critical
+## Success/Failures before passing/warning/critical
 
-In Consul 1.7.0 and later, a check may be configured to become passing/critical
+In Consul 1.7.0 and later, a check may be configured to become passing/warning/critical
```

Comment:
I think we'll need to clarify this a bit, since only passing and critical are available in 1.7.0, and warning will be in 1.11.

For most of the settings we have a list of fields. If we did that here I think that would make it easier to document when the fields were added.

Maybe something like this?

> To prevent flapping health checks, and limit the load they cause on the cluster, a health check may be configured to become passing/warning/critical only after a specified of consecutive checks return passing/critical. The status will not transition states until the configured threshold is reached.

> - `success_before_passing` - ... Added in Consul 1.7.0.
> - `failures_before_warning` - ...  Added in Consul 1.11.0.
> - `failures_before_critical` - ... Added in Consul 1.7.0.

The text for these `...` can probably be copied from the API docs.

The text you added below (about the default value) should probably move into this list of fields section.

Reply:
Yeah, that makes sense. Sure.